src/bootstrap/src/core/config/tests.rs

```diff
@@ -66,6 +66,9 @@ fn detect_src_and_out() {
 
         // test if build-dir was manually given in config.toml
         if let Some(custom_build_dir) = build_dir {
+            // FIXME(#129188): this assertion fails on native Windows because if the rust checkout
```

Comment:
Is the FIXME needed, since https://github.com/rust-lang/rust/issues/129188 is already up?

Reply:
No longer needed

Reply:
Removed.